### ISSUE TITLE
Empty model gets a default value of 0 

### ISSUE THREAD

**Issue by joeappleton18** [vuejs-tips/v-money#28]
Is there any way to override the empty model of being assigned a default value of 0?   I'd  like to just have a blank text box

**Comment by MaGaKZ** [comment]
Yep, because default value is 0, the placeholder is not showing

**Comment by renatosistemasvc** [comment]
me too
I want you to provide the user with the option to leave the field blank

**Comment by bart** [comment]
@nholden This is a different use case. You plugin just set's the input field blank, but we also need a blank value here.

**Comment by nholden** [comment]
Good point, @bart! My PR allows the text box to be blank, but it still results in a value of 0. That worked fine for my use case, but maybe there are situations when you'd want that value to be `null`.

I'm not planning to make any additional changes to my PR, but anyone else is welcome to give it a shot.
